src/main/pages/che-7/end-user-guide/proc_defining-maven-repositories-in-the-settings-xml-file.adoc

```diff
@@ -0,0 +1,102 @@
+// Module included in the following assemblies:
+//
+// using-maven-artifact-repositories
+
+[id="defining-repositories-in-settings-xml_{context}"]
+= Defining repositories in `settings.xml`
+
+You can use the `settings.xml` file, to specify your own artifact repositories at `example.server.org`,
+To do that, ensure, that the `settings.xml` is present in all the containers that use Maven tools, in particular the Maven container and the Java plugin container.
+
+By default, `settings.xml` is located at the `<home dir>/.m2` directory, but because the user home directory is not on a persistent volume, each time you restart workplace, you must re-create the file. This is a known issue, see link:https://github.com/eclipse/che/issues/13318[the issue on GitHub].
```

Comment:
```suggestion
By default, `settings.xml` is located at the `__<home dir>__/.m2` directory. Because the user home directory is not on a persistent volume, you must re-create the file each time you restart the workspace. This is a known issue, see link:https://github.com/eclipse/che/issues/13318[the issue on GitHub].
```
* Or perhaps just use `~/.m2`?
* Not entirely sure whether my `workplace -> workspace` suggestion is warranted here... Maybe confirm, please?

Reply:
Ah, based on the next para, workspace is pbbly correct.